### ISSUE TITLE
What is the real_labels that used to calculate nmi, ari, fscore, adjacc in pretraining stage?

### ISSUE THREAD

**Issue by Vickeyhw** [bytedance/ibot#7]

**Comment by shallowtoil** [comment]
Hi,

`real_labels` is the label ground-truth for each image, which is available for datasets like ImageNet-1K and ImageNet-22K. It is used to monitor the pre-training stability in our early trials (i.e., whether the model is collapsed or not). We keep it to facilitate future research. For datasets without ground-truth labels, you should remove it and this will not change the final results.
